docker/release.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -ex
 
-vt_base_version='v15.0.1'
+vt_base_version='15.0.2-SNAPSHOT'
```

Comment:
@frouioui I believe this needs to be `v15.0.2`. Can you either confirm, or tell me why the format has been changed?

Reply:
This is correct.